frontend/src/components/mine/ContactInfo/PartyRelationships/DefaultContact.js

```diff
@@ -12,71 +14,133 @@ const propTypes = {
-        </p>
-        <br />
-      </div>
-      <div>
-        <Button
-          key={`${props.partyRelationship.mine_party_appt_guid}_edit`}
-          ghost
-          type="primary"
-          onClick={() =>
-            props.openEditPartyRelationshipModal(
-              props.partyRelationship,
-              props.onSubmitEditPartyRelationship,
-              props.handleChange,
-              props.mine
-            )
-          }
-        >
-          <img className="padding-small" src={BRAND_PENCIL} />
-        </Button>
-        <Popconfirm
-          key={`${props.partyRelationship.mine_party_appt_guid}_delete`}
-          placement="topLeft"
-          title={`Are you sure you want to delete this ${props.partyRelationshipTypeLabel}?`}
-          onConfirm={(event) =>
-            props.removePartyRelationship(event, props.partyRelationship.mine_party_appt_guid)
-          }
-          okText="Delete"
-          cancelText="Cancel"
-        >
-          <Button ghost type="primary">
-            <Icon type="minus-circle" theme="outlined" />
-          </Button>
-        </Popconfirm>
-      </div>
-    </div>
-    <h5 className="bold">{props.partyRelationship.party.name}</h5>
-    <p>
-      <Icon type="mail" />
-      &nbsp;&nbsp;
-      {props.partyRelationship.party.email}&nbsp;&nbsp;&nbsp;&nbsp;
-      <br />
-      <Icon type="phone" />
-      &nbsp;&nbsp;
-      {props.partyRelationship.party.phone_no}{" "}
-      {props.partyRelationship.party.phone_ext ? `x${props.partyRelationship.party.phone_ext}` : ""}
-    </p>
-  </div>
-);
+export class DefaultContact extends Component {
+  state = {
+    redirectToRelationshipProfile: false,
+    redirectToPartyProfile: false,
+  };
+
+  render() {
+    if (this.state.redirectToRelationshipProfile === true) {
+      return (
+        <Redirect
+          to={router.RELATIONSHIP_PROFILE.dynamicRoute(
+            this.props.mine.guid,
+            this.props.partyRelationship.mine_party_appt_type_code
+          )}
+        />
+      );
+    }
+    if (this.state.redirectToPartyProfile === true) {
+      return (
+        <Redirect
+          to={router.PARTY_PROFILE.dynamicRoute(this.props.partyRelationship.party.party_guid)}
+        />
+      );
+    }
+
+    return (
+      <Card
+        headStyle={{
+          background: "#EEEEEE",
+          borderTop: "1px solid #CCCCCC",
+          borderRight: "1px solid #CCCCCC",
+          borderLeft: "1px solid #CCCCCC",
+        }}
+        bodyStyle={{
+          borderBottom: "4px solid #CCCCCC",
+          borderRight: "1px solid #CCCCCC",
+          borderLeft: "1px solid #CCCCCC",
+        }}
+        title={
+          <div className="inline-flex between wrap">
+            <div>
+              <h3>{this.props.partyRelationshipTypeLabel}</h3>
+              <p>
+                <Icon type="clock-circle" />
+                &nbsp;&nbsp;
+                {this.props.partyRelationship.start_date === "None"
```

Comment:
Are we coercing these somewhere on the client side or are they coming from the API as the string "None"?

Reply:
They are coming from the API with "None", which I'm not sure is ideal but seems common across our API right now.

Reply:
Hmm... It seems to me that those should definitely be `null` instead. `"None"` seems like the result of not doing a presence check before converting a nullable value to a string.

This is, to the best of my memory, the first time I've come across this in our API.

Reply:
@Jsyro Can you take a look?

Reply:
My understanding of this is that if you return the str(some variable) in python and that variable happens to be None, the string None is returned. Since all of the models have a JSON function that returns the str() of its values any None values are then returned as the string "None".

Reply:
The values in a `json` method dict are only converted to strings if we call `str` on them. We don't have to coerce the value to a string if it's `None`.

A really naive implementation like this should take care of that.
```python
some_attr: self.some_attr is None ? self.some_attr : str(self.some_attr)
```

I expect we want our API returning expected JSON values (ie. `null`), not python-specific type coercion values (`"None"`).

For example:
https://github.com/bcgov/mds/blob/develop/python-backend/app/api/mines/mine/models/mine_type_detail.py#L39

One of `self.mine_disturbance_code` and `self.mine_commodity_code` will always be `None`, and they get sent via the API as `null` because they aren't being coerced to a string.

Reply:
@NWCalvank strings do not need to be converted with str(), as they are serializable. However uuid and dates are not serializable, so we are using str() to convert them to strings. We can do one of 2 things in the future, a) make a str helper method that handles None to return None, or '', or b) see if there is someway of helping python serialize times/uuids (I doubt this exists, i haven't done any research).

Reply:
For now we have made this endpoint return '' instead of "None" above were longer term solutions that can be used across the app. I think we all agree "None" representing null is not what we want.